charts/opentelemetry-collector/README.md

```diff
@@ -138,6 +138,25 @@ config:
           - otlphttp
 ```
 
+### Configuration for Kubernetes events
+
+The collector can be used to collect Kubernetes Events as logs.
+This feature is disabled by default. It has the following requirements:
+
+- It requires the [contrib](https://github.com/open-telemetry/opentelemetry-collector-contrib) version
```

Comment:
Technically any distribution of the collector with the k8s_events receiver should work.